examples/migration/CreateCompleteCampaignAdWordsApiOnly.py

```diff
@@ -0,0 +1,202 @@
+"""This example creates serach campaign with the help of Google-Ads Api"""
+
+
+"""This code example is the first in a series of code examples that shows how to create
+a Search campaign using the AdWords API, and then migrate it to Google Ads API one
+functionality at a time. See other examples in this directory for code examples in various
+stages of migration.
+
+This code example represents the initial state, where the AdWords API is used to create a
+campaign budget, a Search campaign, ad groups, keywords and expanded text ads. None of the
+functionality has yet been migrated to the Google Ads API.
+"""
+
+import datetime
+import uuid
+from googleads import adwords
+#Number of ads being added/updated in this code example.
+NUMBER_OF_ADS = 5
+#The list of keywords being added in this code example.
+KEYWORDS_TO_ADD = ["mars cruise", "space hotel" ]
+import urllib.parse
+
+
+def createCampaignBudget(client):
+   # Create a budget, which can be shared by multiple campaigns.
+    budget_service = client.GetService('BudgetService', version='v201809')
+    budget = {
+        'name' : 'Interplanetary Cruise Budget #%s' % uuid.uuid4(),
+        'amount': {
+            'microAmount' : '50000000'
+        },
+        'deliveryMethod' : 'STANDARD'
+    }
+    budget_operations = [{
+        'operator': 'ADD',
+        'operand': budget
+    }]
+    # Add budget.
+    results = budget_service.mutate(budget_operations)
+    createdBudget = results['value'][0]
+    print("Budget with ID {} and name {} was created".format(\
+           createdBudget['budgetId'],createdBudget['name']))
+    return createdBudget['budgetId']
+
+
+def createCampaign(client, budgetId):
+    campaign_service = client.GetService('CampaignService', version='v201809')
+    campaign = {
+        'name': 'Interplanetary Cruise #%s' % uuid.uuid4(),
+        'advertisingChannelType': 'SEARCH',
+        # Recommendation: Set the campaign to PAUSED when creating it to stop the
+        # ads from immediately serving. Set to ENABLED once you've added
+        # targeting and the ads are ready to serve.
+        'status': 'PAUSED',
+        'biddingStrategyConfiguration': {
+            'biddingStrategyType': 'MANUAL_CPC',
+        },
+        'startDate': (datetime.datetime.now() +
+        datetime.timedelta(1)).strftime('%Y%m%d'),
+        'endDate': (datetime.datetime.now() +
+        datetime.timedelta(365)).strftime('%Y%m%d'),
+        # Budget (required) - note only the budget ID is required.
+        'budget': {
+            'budgetId': budgetId
+        },
+        'networkSetting': {
+            'targetGoogleSearch': 'true',
+            'targetSearchNetwork': 'true',
+        }
+    }
+    campaign_operations = [{
+        'operator': 'ADD',
+        'operand': campaign
+    }]
+    results = campaign_service.mutate(campaign_operations)
+    createdCampaign = results['value'][0]
+    print("CreatedCampign with ID {} and name {} was created". \
```

Comment:
Don't continue the statement like this, try to continue after an open paren, so like 

```python
print("CreatedCampign with ID {} and name {} was created".format(
    ...
```